boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -71,11 +68,7 @@ REGISTRY_TOKEN ?=
 
 BINFILE=build/_output/bin/$(OPERATOR_NAME)
 MAINPACKAGE = ./
-API_DIR = $(NEW_API_DIR)
-ifeq ($(USE_OLD_SDK), TRUE)
-MAINPACKAGE = ./cmd/manager
-API_DIR = ./pkg/apis
-endif
+API_DIR = ./api
```

Comment:
Or just inline it. Is it used anywhere else?

Reply:
All three of these, actually.

Unless they're overridden in consuming projects?

Reply:
They shouldn't be overridden since we should all have the same structure, so +1